### DISCUSSION TITLE
Is there a way to make a blog overview like material's blog?

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3243] [Q&A] [answered]
Recently did I take a look at the blog of squidfunk at https://squidfunk.github.io/mkdocs-material/blog/ and I love the way it looks.

This inspired me to try and update [my own blog](https://github.com/Andre601/blog) made with MkDocs, but taking a look at the markdown page of the blog tells me that this seems to be a manual labor you have to do each time a blog post is published.

While there are plugins for MkDocs that allow easier blogging do most of them have limitations I like to avoid such as very rigid folder structures or no real customization options.
The most promising one seems to be [liang2kl/mkdocs-blogging-plugin](https://github.com/liang2kl/mkdocs-blogging-plugin) as it allows customization of the template used.

With that said do I have no real experience with jinja2 templating and don't want to risk breaking everything...
I don't even know if using this plugin over my current setup would preserve the current URLs or not.
And since one of the limitations seems to be that it can't be used on windows (assuming I can't see it with `mkdocs serve`) is this more tedious to work with.

So if anyone has experience with that plugin and is willing to help me out here, I would really appreciate it a lot...

**Comment by squidfunk** [comment]
Yes, it's currently manual labor. You can take a look at #3192 which seems to abstract away some stuff.

> While there are plugins for MkDocs that allow easier blogging do most of them have limitations I like to avoid such as very rigid folder structures or no real customization options.

I'd be very interested in learning more about that, especially where the plugins fall short.

**Comment by Andre601** [comment, marked as answer]
I managed to create something similar to what you have using the [mkdocs-blogging-plugin](https://github.com/liang2kl/mkdocs-blogging-plugin) which works very nicely.

One downside is, that stuff like svg icons need to be added in a separate folder for the `{% include %}` to work (Or requires you to define an absolute path rather than relative).

But I see this as a rather small downside compared to what you gain.

Either way, this is, what I currently have now:
![image](https://user-images.githubusercontent.com/11576465/146182978-1e20fbf9-dbee-493c-bad5-d1226427dadf.png)

The template override I have is here:
https://github.com/Andre601/blog/blob/master/theme/blog/blog-override.html

**Reply by squidfunk** [reply]
I've started to collect some ideas in #3353. Any input is appreciated, feel free to leave a comment!
